packages/client/src/components/Input/index.tsx

```diff
@@ -30,9 +30,22 @@ export const Input: FC<InputProps> = (props) => {
 					name={name}
 					onChange={onChange}
 					onBlur={onBlurHandler}
+					data-errors={errors}
 				/>
 				<div className='underLine'></div>
 			</div>
+			{
+				name==='login' && errors && errors['login']  &&
+					<p className='formError'>{errors['login']}</p>
+			}
```

Comment:
Прости за "духоту" тут можно использовать само имя в качестве ключа к объекту `error`, например вот так:
```suggestion
			{
				errors && errors[name]  &&
					<p className='formError'>{errors[name]}</p>
			}
```

Тогда не нужно будет добавлять дополнительный строки при появлении других имен инпутов 🙈

Это не критично, исправлять не обязательно